build.gradle.kts

```diff
@@ -79,49 +79,45 @@ dependencies {
     compileOnly("org.apache.commons:commons-lang3:3.3.2")
     compileOnly("org.jetbrains:annotations:23.0.0")
 
-    compileOnly("gg.essential:vigilance-1.8.9-forge:+") {
+    compileOnly(libs.vigilance) {
         attributes { attribute(common, true) }
         isTransitive = false
     }
 
-    compileOnly("cc.polyfrost:universalcraft-1.8.9-forge:246") {
+    compileOnly("cc.polyfrost:universalcraft-1.8.9-forge:${libs.versions.universalcraft.get()}") {
```

Comment:
shouldn't it just be a library and not a version? like `compileOnly(libs.universalcraft)`

Reply:
we need to do this for the platform buildscript so it doesn't matter